conan.cmake

```diff
@@ -264,9 +264,13 @@ function(conan_cmake_detect_unix_libcxx result)
         endif()
     endforeach()
 
+    if(APPLE)
+        set(xcode_sysroot_option "--sysroot=${CMAKE_OSX_SYSROOT}")
```

Comment:
Is it possible that CMAKE_OSX_SYSROOT is not defined? For example in iOS? I would say it is possible, and this would break for some users not in OSX. Does it make sense to test for CMAKE_OSX_SYSROOT existence instead?

(sorry this took a while to review, Conan amazing growth keeps the team very busy)

Reply:
I also would suggest to check `CMAKE_OSX_SYSROOT` instead of `APPLE`.
Also @dforsten I have added a test for this issue.